pkg/collect/cluster_resources.go

```diff
@@ -137,6 +137,14 @@ func ClusterResources(c *Collector) (map[string][]byte, error) {
 		return nil, err
 	}
 
+	// crs
+	customResources, crErrors := crs(ctx, crdClient)
+	clusterResourcesOutput["cluster-resources/custom-resources.json"] = customResources
+	clusterResourcesOutput["cluster-resources/custom-resources-errors.json"], err = marshalNonNil(crErrors)
```

Comment:
can we split resources per resource type? I think it would help to have dedicated file per resource type.